index.html

```diff
@@ -339,11 +339,15 @@ <h3>Contact me</h3>
       project that needs coding. I’d
        love to help with it
   </p>
-  <form action="https://formspree.io/f/xyyovzvy" method="post">
-    <input type="text" name="name" placeholder="ex: pamphile" maxlength="30" required> <br>
-    <input type="email" name="email" placeholder="ex: pamphilemkp@gmail.com" maxlength="30" required> <br>
-    <textarea placeholder=" Write your message here" name="comment" id="comment" cols="30" rows="10" required maxlength="500"></textarea><br>
+  <form id="form" action="https://formspree.io/f/xyyovzvy" method="post">
+    <p class="Paragraph-errors errorMessage"></p>
+    <input type="text" name="name" id="name" placeholder="ex: pamphile" maxlength="30" > <br>
+    <input type="email" name="email" id="email" placeholder="ex: pamphilemkp@gmail.com" maxlength="30" >
+    <br>
+    <textarea placeholder=" Write your message here" id="comment" name="comment" cols="30" rows="10"  maxlength="500"></textarea><br>
+    
```

Comment:
- [ **OPTIONAL** ] **Important:** Kindly make sure you update the message text color in the textarea field by using the same(color) as on the Figma design for good accessibility and to match the Figma template. I know you can do it :muscle: :muscle:.